app/views/uploads/_form.html.erb

```diff
@@ -13,14 +13,9 @@
       </div>
 
       <div class="form-group">
-        <%= f.label(:column_separator, for: "upload_col_sep") %><br />
-        <%= f.select(:col_sep, options_for_select(Settings.csv_upload.column_separators, @upload.col_sep),
-                     { include_blank: "Select column separator" },
-                     { class: "form-control", required: true }) %>
-      </div>
-
-      <div class="form-group">
-        <%= f.label(:upload_file) %>
+        <%= f.label(:upload_file) do
+          raw('Upload file <br><span class="not-bold">Valid file formats include: .txt, .csv</span>')
+        end %>
```

Comment:
Alternative styling, using Bootstrap:
```
        <%= f.label(:upload_file) %>
        <p class="help-block">Valid file formats include: .txt, .csv</p>
```
Use whichever you prefer.

Reply:
changed